neusomatic/python/call.py

```diff
@@ -412,7 +411,17 @@ def call_neusomatic(candidates_tsv, ref_file, out_dir, checkpoint, num_threads,
 
     vartype_classes = ['DEL', 'INS', 'NONE', 'SNP']
     data_transform = matrix_transform((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+
+    ensemble = False
+    with open(candidates_tsv[0]) as i_f:
+        for line in i_f:
+            x = line.strip().split()
+            if len(x) == 97:
```

Comment:
@msahraeian you just need to read one line to determine whether ensemble or not, right? If that's the case, the loop can be avoided. You can use the `readline` function, instead.